src/Controls/PlanarControls.js

```diff
@@ -63,22 +66,32 @@ let travelDuration = 0;
 let travelUseRotation = false;
 let travelUseSmooth = false;
 
+// zoom changes (for orthographic camera)
+let startZoom = 0;
+let endZoom = 0;
+
+// ray caster for drag movement
+const rayCaster = new THREE.Raycaster();
+const plane = new THREE.Plane(
+    new THREE.Vector3(0, 0, -1),
```

Comment:
why `-1` for Z? it's doesn't work with `1`?

Reply:
Setting `z` value to `1` generates inconsistent drag movements in `view_25D_map.html` example.